language/tools/move-package/src/resolution/dependency_graph.rs

```diff
@@ -904,3 +945,23 @@ fn str_escape(s: &str) -> Result<String, fmt::Error> {
 fn path_escape(p: &Path) -> Result<String, fmt::Error> {
     str_escape(p.to_str().ok_or(fmt::Error)?)
 }
+
+/// Checks if dependencies of a given package in two different dependency graph maps are the
+/// same.
+fn pkg_deps_equal_new(
```

Comment:
To support the comment below about more robust error message, this function will need to return the delta between the outbound edges in these graphs, rather than just a boolean.

```
fn diff_package_dependencies(
    pkg_name: PM::PackageName,
    graph_a: &DiGraphMap<PM::PackageName, Dependency>,
    graph_b: &DiGraphMap<PM::PackageName, Dependency>,
) -> /* ... not sure what a good return type would be here ... */
```